bin/dev

```diff
@@ -40,9 +42,16 @@ if [ "$USE_TAILSCALE" = "1" ]; then
     echo "Error: tailscale serve failed. On Linux, run once: sudo tailscale set --operator=\$USER" >&2
     exit 1
   fi
-  echo "Login with david@example.com to: https://$TS_HOSTNAME:$TS_PORT/"
+  echo "  Tailscale: https://$TS_HOSTNAME:$TS_PORT/"
 else
-  echo "Login with david@example.com to: http://fizzy.localhost:$PORT/"
+  echo "  Local: http://fizzy.localhost:$PORT/"
 fi
 
-./bin/rails server -p $PORT
+bin/rails runner - <<EOF
+  require 'socket'
+  local_ip = Socket.ip_address_list.find { |ai| ai.ipv4? && !ai.ipv4_loopback? }&.ip_address
+  puts "  Network: http://#{local_ip}:3006/" if local_ip
+EOF
```

Comment:
Would omit. Folks who are doing this will know their IP. Preferably, they'd use Avahi/Bonjour instead of a fixed IP, or they'd just use Tailscale.